vaadin-dialog-flow/src/main/java/com/vaadin/flow/component/dialog/Dialog.java

```diff
@@ -433,6 +438,10 @@ private void attachComponentRenderer() {
                 "<flow-component-renderer appid=\"%s\" nodeid=\"%s\"></flow-component-renderer>",
                 appId, nodeId);
         template.setProperty("innerHTML", renderer);
-    }
 
+        getElement().executeJs("this.$.overlay.$.overlay.style.height=$0",
```

Comment:
Discussed this in person.`setWidth/Height` would produce "24px" (Lumo padding) smaller outcome with this approach alone.

In order not to break the existing functionality, we'll also need to add the effective padding size to the value applied to the element. But this is only the case with non-relative units; `setWidth(100%)` should result in full width, not full-width + padding.

Reply:
New decision related to this, we'll instead go with the current revision of this PR and consider the new functionality as fix for https://github.com/vaadin/vaadin-dialog-flow/issues/144